apps/web/components/sections/project/about-project.tsx

```
@@ -0,0 +1,94 @@
+          ))}
+        </ul>
+      </div>
+
+      <div className="border-t border-gray-200 my-4"></div>
+
+      <div className="latest-updates">
+        <h3 className="text-xl font-bold text-black mb-2">Latest Updates</h3>
+        <ul>
+          {updates.map((update, index) => (
+            <li
+              key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)